src/scantree/__init__.py

```diff
@@ -1,19 +1,7 @@
-from __future__ import print_function, division
-
-from ._path import (
-    RecursionPath,
-    DirEntryReplacement
-)
-from ._node import (
-    DirNode,
-    LinkedDir,
-    CyclicLinkedDir
-)
-from ._filter import RecursionFilter
-from ._scan import (
-    scantree,
-    SymlinkRecursionError
-)
-
 from . import _version
-__version__ = _version.get_versions()['version']
+from ._filter import RecursionFilter  # noqa: F401
```

Comment:
there are bunch of unused imports here (at least unused inside this file
i guess there is a way to improve it but i'd leave it to the next PR

Reply:
Yes, this is intentional (this is where they should be imported from), I have missed adding them to `__all__` No worries I will fix!